pkg/db-repo/change_history_manager.go

```diff
@@ -155,24 +155,19 @@ func (c *ChangeHistoryManager) executeMigration(originalTable, historyTable *tab
 
 	if !historyTable.exists {
 		statements = append(statements, c.createHistoryTable(historyTable))
-		statements = append(statements, c.dropHistoryTriggers(originalTable, historyTable)...)
-		statements = append(statements, c.createHistoryTriggers(originalTable, historyTable)...)
 		c.logger.Info("creating change history setup")
-		return c.execute(statements)
-	}
-
-	updated, statement := c.updateHistoryTable(historyTable)
-	if updated {
+	} else if updated, statement := c.updateHistoryTable(historyTable); updated {
+		// if the table exists, we would "update" it with the same columns - skip it
 		statements = append(statements, statement)
-		statements = append(statements, c.dropHistoryTriggers(originalTable, historyTable)...)
-		statements = append(statements, c.createHistoryTriggers(originalTable, historyTable)...)
 		c.logger.Info("updating change history setup")
-		return c.execute(statements)
+	} else {
+		c.logger.Info("change history setup was already up to date")
```

Comment:
Are you missing a `return nil` here? Or do we want to always drop and recreate the triggers? Wouldn't that run the risk that a task starting up drops the triggers, some other task does something, and we miss some update?

Reply:
Indeed we want to always recreate the triggers.
The migration is run during the initial start up of the app, it won't be healthy without being complete. So it shouldn't be a problem.